vms/platformvm/state/state_test.go

```diff
@@ -657,7 +318,7 @@ func TestValidatorWeightDiff(t *testing.T) {
 }
 
 // Tests PutCurrentValidator, DeleteCurrentValidator, GetCurrentValidator,
-// GetValidatorWeightDiffs, GetValidatorPublicKeyDiffs
+// ApplyValidatorWeightDiffs, ApplyValidatorPublicKeyDiffs
 func TestStateAddRemoveValidator(t *testing.T) {
```

Comment:
This test should be extended to include delegator changes and more complex validator changes

Reply:
Or maybe extend `TestGetValidatorsSetProperty`  to include delegators too? Although it may become even more complex.